docker-compose.yml

```diff
@@ -48,16 +48,16 @@ services:
                 depends_on:
                         - hub-db
                 environment:
-                - HUB_DB_ENV_MYSQL_DATABASE=hubdb
-                - HUB_DB_ENV_MYSQL_USER=hubdb_user
-                - HUB_DB_ENV_MYSQL_PASSWORD
-                - HUB_DB_PORT_3306_TCP_ADDR=hub-db
-                - HUB_EXT_FILE_URL=http://localhost:8181/sites/default/files
-                - PHP_MEMORY_LIMIT=500M
-                - PHP_UPLOAD_MAX_FILE_SIZE=500M
-                - PHP_POST_MAX_SIZE=500M
-                - SIMPLETEST_DB=mysql://hubdb_user:hubdb_pass@localhost/hubdb
-                - XDEBUG_IP=127.0.0.1 # Replace with your machines IP: $ ipconfig getifaddr en0 
+                 - HUB_DB_ENV_MYSQL_DATABASE=hubdb
+                 - HUB_DB_ENV_MYSQL_USER=hubdb_user
+                 - HUB_DB_ENV_MYSQL_PASSWORD
+                 - HUB_DB_PORT_3306_TCP_ADDR=hub-db
+                 - HUB_EXT_FILE_URL=http://localhost:8181/sites/default/files
+                 - PHP_MEMORY_LIMIT=500M
```

Comment:
I'd like the PHP_MEMORY_LIMIT to match what is in production to catch memory issues quicker.  Both prod sites use 256M